### PR TITLE
Update readme.txt and README.md

### PR DESCRIPTION
**Pull request by goldenapples**
Adds sections about how to use the plugin, how we use it at Fusion, and data migration.

See #9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +25,116 @@ easily filtered in themes, plugins and templates, and since the original
 attachment data is preseved as attributes on the shortcode, it becomes much
 easier for modify the way images are marked up in themes.
 
-For best results, use with [Shortcake (Shortcode UI)](https://github.com/fusioneng/Shortcake) plugin.
+For best results, use this with the [Shortcake (Shortcode
+UI)](https://github.com/fusioneng/Shortcake) plugin. Shortcake offers an
+easy to use interface to manage shortcodes in post content.
+
+
+## Customizing Output ##
```
Comment (danielbachhuber) [inline README.md]: Can you call this subhead `Installation` so it appears on WordPress.org?

File: README.md [hunk 2]
```diff
@@ -26,7 +25,116 @@ easily filtered in themes, plugins and templates, and since the original
...
+
+* **Inline sharing buttons**. We've added share links to each of the
+  images on our site. Because these are inserted through a ilter on
+  a shortcode and not in the post content, it's easy to modify them on the
+  fly. And having this logic in template files rather in on-page javascript
+  that runs after page load makes it quicker for users.
+
+* **Photo credits**. We've added "credit" as an image meta field, and we
+  use a filter on 'img_shortcode_output_after_linkify' to display it on all
+  images.
```
Comment (danielbachhuber) [inline README.md]: Good! Might be worth moving this examples block to `Description` so it's not hidden.

File: README.md [hunk 3]
```diff
@@ -26,7 +25,116 @@ easily filtered in themes, plugins and templates, and since the original
...
+made.
+
+`wp image-shortcode revert <ids> [--dry-run]`
+
+This command finds all `[img]` shortcodes in the content of any of the
+posts specified in `<ids>`, and replaces them with the markup that would
+be generated by those shortcodes. Note that this runs any filters in your
+theme, so that if you have filtered the output of the shortcodes at any
+output, those filters will be reflected in the coverted post content.
+
```
Comment (danielbachhuber) [inline README.md]: This could be under `Installation` for more prominence.

Also, shouldn't the command be `image-shortcake` ?

**Comment by danielbachhuber** [comment]
Looks pretty good — left some comments.

**Comment by goldenapples** [comment]
Good catches. The cli command name makes much more sense. I'll rename the `revert` command that I'm working on in #31 to follow the same conventions.

**Comment by danielbachhuber** [comment]
:cake:

**Pull request merged**